### ISSUE TITLE
stubgen -m/-p "Failed to import xx skipping it" because of ModuleNotFoundError

### ISSUE THREAD

**Issue by xbian0403** [python/mypy#6794]
`stubgen -m my_module`
`stubgen -p my_package `
output: "Failed to import my_module/my_package skipping it"  Error without details.
Because the environment actually missing packages which generate ModuleNotFoundError.

mypy 0.701
python3.7

**Comment by ilevkivskyi** [comment]
Could you please give some more details? I am not sure what exactly here is the bug.

**Comment by gvanrossum** [comment]
@xbian0403 demonstrated this to me. The problem was that the module imports another module that's not installed. When using `stubgen -m` (or `stubgen -p`) it tries to import the module (or package) and this import fails due to the missing dependency. However if you point stubgen to the file (e.g. `stubgen some/dir/my_module.py`) it works, it just ignores the bad import.

We spent some time debugging what was going on -- it would have been quicker if stubgen had given us the text of the exception rather than just saying "Failed to import xx". (Note that `xx` was the name of the module we pointed stubgen at, not name of the dependency that wasn't installed.)
